tests/assets/vott_json_dataset/dataset/train-export.json

```diff
@@ -0,0 +1,206 @@
+{
+    "name": "test",
+    "videoSettings": {
+        "frameExtractionRate": 15
+    },
+    "tags": [
+        {
+            "name": "animal",
+            "color": "#5db300"
+        },
+        {
+            "name": "dog",
+            "color": "#e81123"
+        },
+        {
+            "name": "person",
+            "color": "#6917aa"
+        }
+    ],
+    "useSecurityToken": true,
+    "securityToken": "test Token",
+    "id": "0OPxbBiQE",
+    "activeLearningSettings": {
+        "autoDetect": false,
+        "predictTag": true,
+        "modelPathType": "coco"
+    },
+    "version": "2.2.0",
+    "lastVisitedAssetId": "0d3de147fea94f1797f9d012acad9666",
+    "assets": {
+        "0d3de147f": {
+            "asset": {
+                "format": "jpg",
+                "id": "0d3de147f",
+                "name": "img0001.jpg",
+                "path": "tests/assets/vott_json_dataset/dataset/img0001.jpg",
```

Comment:
Probably, this path should not contain the leading directories.

Reply:
What for? If I remove the leading directories, it will break.

Reply:
The `tests/assets/vott_json_dataset/` part lies outside the dataset root directory, so it must not be be a part of a dataset, it must be computed during loading. Currently it just stores some private information.

Reply:
Fixed.